### PR TITLE
design(1314): add `--sudo` option

### PR DESCRIPTION
**Pull request by kkokufud**
## Context
Update sd-local design docs for adding `--sudo` option.

## References
https://github.com/screwdriver-cd/sd-local/pull/21

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: design/sd-local.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ test
 
 ### Prerequisites
 - Docker runtime
+  - Have docker runtime execute permission (or you can use `--sudo` option)
```
Comment (MysticDoll) [inline design/sd-local.md]: How about `Permission for docker runtime`?

**Comment by yuichi10** [comment]
Could you add update status?

**Comment by yuichi10** [comment]
It looks this PR failed with eslint error. Could you fix that error too?
Thank you

File: design/sd-local.md (hunk 1 above)
Comment (kkokufud) [reply design/sd-local.md]: I fixed.

**Comment by kkokufud** [comment]
> Could you add update status?
I fixed.

**Comment by sakka2** [comment]
> It looks this PR failed with eslint error. Could you fix that error too?
> Thank you

We are asking about this error to the OSS team.
https://screwdriver-cd.slack.com/archives/C1T5T67PT/p1584332486043600

**Comment by jithine** [comment]
@sakka2 This is a valid error. Looks like related to https://www.npmjs.com/package/eslint-config-airbnb-base/v/14.1.0 release 3 days ago

Can be fixed by running this locally

```./node_modules/.bin/eslint . --fix```

This is the change, can you please fix it with this PR itself

```--- a/features/support/world.js
+++ b/features/support/world.js
@@ -5,7 +5,7 @@ const path = require('path');
 const env = require('node-env-file');
 const requestretry = require('requestretry');
 const { setWorldConstructor } = require('cucumber');
-const request = require('../support/request');
+const request = require('./request');
```

**Comment by sakka2** [comment]
@jithine 
Thanks! I fixed lint error.

**Pull request merged**